tests/tas-mtls/src/test/java/org/alfresco/rest/MtlsRestTest.java

```diff
@@ -85,4 +94,24 @@ public void checkIfMtlsIsEnabledForSharedFileStorage()
     {
         Assert.assertThrows(SSLHandshakeException.class, () -> client.execute(new HttpGet("https://localhost:8099")));
     }
+
+    @Test
+    public void checkIfMtlsIsEnabledForSearchEngine()
+    {
+        Assert.assertThrows(SSLHandshakeException.class, () -> client.execute(new HttpGet("https://localhost:8083/solr")));
+    }
+
```

Comment:
Method selectSharedFolder has been created as a work around to the restClient kind of not having functionality to create a folder, or at least I've failed to use it. Wherever a folder is created in tests it's done through DataContent and since I can't use it with sslContext (its code is in an archived repository, it's hard to introduce changes).